src/main.js

```diff
@@ -130,6 +130,7 @@ takeBackBtn.addEventListener("click", goToMain);
 backToMainBtn.addEventListener("click", goToMain);
 showMadePosterBtn.addEventListener("click", displayMadePoster);
 savePosterBtn.addEventListener("click", savePosterToGrid);
+savedPostersGrid.addEventListener("dblclick", deletePoster);
```

Comment:
Nitpick: I might name this `savedPoster` since we are referring to the individual poster and not the whole grid of posters.